packages/headless/src/features/search/search-mappings.ts

```diff
@@ -61,9 +65,9 @@ function mapFacetRequest(
     mappings.dateFacetValueMap[facetId] = {};
     return {
       ...facetRequest,
-      currentValues: currentValues.map((value) =>
-        mapDateRangeRequest(value, facetId, mappings)
-      ),
+      currentValues: currentValues
```

Comment:
Here anytime a user inputs an invalid value (e.g. through the URL manager) it will remove it from the request, failing silently.